src/bootstrap/bootstrap.py

```diff
@@ -1035,6 +1035,10 @@ def check_vendored_status(self):
                 eprint('       Run `cargo vendor {}` to initialize the '
                       'vendor directory.'.format(sync_dirs))
                 eprint('Alternatively, use the pre-vendored `rustc-src` dist component.')
+                eprint('You can download it from '
+                    'https://forge.rust-lang.org/infra/other-installation-methods.html#source-code')
+                eprint('and then place the vendor directory from the archive in the root')
+                eprint('of the rust project.')
```

Comment:
Thank you for your contribution!

We want to refer to commit-specific sources rather than lates nightly/beta/master. See https://github.com/rust-lang/rust/issues/110163#issuecomment-1502647562

Reply:
I'm fine with changing that, but like I wrote in the PR description, the original issue mentioned that this download is not documented anywhere. Shouldn't it be documented in some better place first?

Reply:
> Shouldn't it be documented in some more appropriate place first?

We don't need to consider that as a blocker; as we can update [forge.rust-lang.org/infra/other-installation-methods](https://forge.rust-lang.org/infra/other-installation-methods.html?highlight=download#other-ways-to-install-rustup) doc anytime.